examples/catalina_attributes.pp

```diff
@@ -1,6 +1,8 @@
+# @param catalina_base
+# @param show_diff
 define tomcat::config::example (
-  $catalina_base     = '/tmp',
-  Boolean $show_diff = true,
+  String[1] $catalina_base = '/tmp',
```

Comment:
This can also be a `Stdlib::Absolutepath`

Reply:
👍